TFT/src/User/Menu/FeatureSettings.c

```diff
@@ -295,79 +442,154 @@ void loadFeatureSettings()
+
     if (item_index < SKEY_COUNT)
     {
       switch (item_index)
       {
-        case SKEY_ACK:
-          settingPage[item_index].icon = toggleitem[infoSettings.terminalACK];
+        case SKEY_STATUS_SCREEN:
+          settingPage[item_index].icon = iconToggle[infoSettings.status_screen];
           break;
 
-        case SKEY_INVERT_X:
-          settingPage[item_index].icon = toggleitem[infoSettings.invert_axis[X_AXIS]];
+        case SKEY_TERMINAL_ACK:
+          settingPage[item_index].icon = iconToggle[infoSettings.terminalACK];
           break;
 
-        case SKEY_INVERT_Y:
-          settingPage[item_index].icon = toggleitem[infoSettings.invert_axis[Y_AXIS]];
+        case SKEY_INVERT_X_AXIS:
+          settingPage[item_index].icon = iconToggle[infoSettings.invert_axis[X_AXIS]];
           break;
 
-        case SKEY_INVERT_Z:
-          settingPage[item_index].icon = toggleitem[infoSettings.invert_axis[Z_AXIS]];
+        case SKEY_INVERT_Y_AXIS:
+          settingPage[item_index].icon = iconToggle[infoSettings.invert_axis[Y_AXIS]];
           break;
 
-        #ifdef PS_ON_PIN
-          case SKEY_POWER:
-            settingPage[item_index].icon = toggleitem[infoSettings.auto_off];
+        case SKEY_INVERT_Z_AXIS:
+          settingPage[item_index].icon = iconToggle[infoSettings.invert_axis[Z_AXIS]];
+          break;
+
+        case SKEY_PERSISTENT_INFO:
+          settingPage[item_index].icon = iconToggle[infoSettings.persistent_info];
+          break;
+
+        case SKEY_FILE_LIST_MODE:
+          settingPage[item_index].icon = iconToggle[infoSettings.file_listmode];
+          break;
+
+        case SKEY_ACK_NOTIFICATION:
+          setDynamicTextValue(i, (char *)itemNotificationType[infoSettings.ack_notification]);
+          break;
+
+        case SKEY_DEFAULT_MODE:
+          setDynamicTextValue(i, (char *)itemDefaultMode[infoSettings.mode]);
+          break;
+
+        case SKEY_SERIAL_ALWAYS_ON:
+          settingPage[item_index].icon = iconToggle[infoSettings.serial_alwaysOn];
+          break;
+
+        #ifdef ST7920_SPI
+          case SKEY_MARLIN_SHOW_TITLE:
+            settingPage[item_index].icon = iconToggle[infoSettings.marlin_mode_showtitle];
             break;
-        #endif
 
-        #ifdef FIL_RUNOUT_PIN
-          case SKEY_RUNOUT:
-            settingPage[item_index].valueLabel = itemRunout[infoSettings.runout];
+          case SKEY_MARLIN_FULLSCREEN:
+            settingPage[item_index].icon = iconToggle[infoSettings.marlin_mode_fullscreen];
+            break;
+
+          case SKEY_MARLIN_TYPE:
+            setDynamicTextValue(i, (char *)itemMarlinType[infoSettings.marlin_type]);
             break;
         #endif
 
+        case SKEY_HEATED_BED:
+          settingPage[item_index].icon = iconToggle[infoSettings.bed_en];
+          break;
+
+        case SKEY_HEATED_CHAMBER:
+          settingPage[item_index].icon = iconToggle[infoSettings.chamber_en];
+          break;
+
```

Comment:
there is no point in placing these in menu. these are already being set through config file.

Reply:
entries removed